smarts/core/renderer.py

```diff
@@ -47,25 +48,59 @@
 from .coordinates import Pose
 from .masks import RenderMasks
 from .scenario import Scenario
+from .utils.config import SmartsConfig
+
+
+def get_renderer(sim_id, config: SmartsConfig):
+    """Get the appropriate renderer given the simulation id and config.
+    Args:
+        sim_id (str): The id of a SMARTS simulation the renderer should be related to.
+        config (SmartsConfig): A configuration file determining extra configuration.
+
+    Returns:
+        Renderer: A new renderer based on the given configuration.
+    """
+    _renderer = Renderer(sim_id, config)
+    return _renderer
```

Comment:
Given that it's just a local variable, I'm not sure why the underscore prefix was used for `_renderer` although it's of course fine.

Reply:
Removed this utility for now.